src/scripts/util.js

```
@@ -327,18 +260,19 @@ Date.prototype.getWeek = function () {
     d = new Date(Date.UTC(d.getFullYear(), d.getMonth(), d.getDate()))
     d.setUTCDate(d.getUTCDate() + 4 - (d.getUTCDay() || 7))
     const yearStart = new Date(Date.UTC(d.getUTCFullYear(), 0, 1)),
-        weekNo = Math.ceil((((d - yearStart) / 86400000) + 1) / 7)
+        weekNo = Math.ceil((((d.getTime() - yearStart.getTime()) / 86400000) + 1) / 7)
     return weekNo
 }
 
+Date.prototype.getHoursWithDecimals = function () { return this.getHours() + (this.getMinutes() / 60) }
+
 Date.prototype.getFormattedDay = function () {
     let d = this
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)